### PR TITLE
TabView gap between active tab and active content fix

### PR DESCRIPTION
**Pull request by ojhad**
## Description
Gap between active tab and active tab content when device scale factor is not 100%
image

The `SelectedBackgroundPath` path is not being drawn all the way down in non-100% scale factors. Layout rounding API does not seem to fix this. I added an extra pixel to `TabGeometry` which is being used to draw the background. That fixed the drawing at non-100% scale factors while still looking correct at 100% scale.

# Motivation and Context
fixes internal bugs 37332027

# Screenshots

Pre-Fix on top of Post-Fix

100% scale
image
image
image
image

125% scale
image
image
image
image

150% scale
image
image
image
image

175% scale
image
image
image
image

200% scale
image
image
image
image

### CODE REVIEW

File: dev/TabView/TabView.xaml [hunk 1]
```diff
@@ -40,7 +40,8 @@
 
                         <Grid x:Name="TabContainerGrid"
                                 Background="{TemplateBinding Background}"
-                                XYFocusKeyboardNavigation="Enabled">
+                                XYFocusKeyboardNavigation="Enabled"
+                                Margin="0,0,0,-1">
```
Comment (RBrid) [inline dev/TabView/TabView.xaml:44]: That fix seems wrong to me. If the contour line (LeftBottomBorderLine, BottomBorderLine, RightBottomBorderLine) is semi-transparent, the TabContentPresenter will show through, no?

Comment (StephenLPeters) [reply dev/TabView/TabView.xaml:44]: Agreed, This is also a pixel snapping issue, a negative margin is not appropriate at all scale factors.

**Review by RBrid — APPROVED** [review]

File: dev/TabView/TestUI/TabViewPage.xaml.cs [hunk 2]
```diff
@@ -409,6 +419,96 @@ private void ShortLongTextButton_Click(object sender, RoutedEventArgs e)
...
+            var redBrush = new SolidColorBrush();
+            redBrush.Color = Colors.Red;
+            BackgroundGrid.Background = redBrush;
+
+            var tabBrush = new SolidColorBrush();
+            tabBrush.Color = Colors.Blue;
+            SetFirstTabSelectedBackgroundPathFill(tabBrush);
+
+            var contentBrush = new SolidColorBrush();
+            contentBrush.Color = Colors.Green;
```
Comment (StephenLPeters) [inline dev/TabView/TestUI/TabViewPage.xaml.cs:434]: should these colors be translucent?

Comment (StephenLPeters) [reply dev/TabView/TestUI/TabViewPage.xaml.cs:434]: Or maybe have a button to set translucency to 50%

**Review by StephenLPeters — APPROVED** [review]

File: dev/TabView/TestUI/TabViewPage.xaml.cs (hunk 2 above)
Comment (ojhad) [reply dev/TabView/TestUI/TabViewPage.xaml.cs:434]: I added buttons to set each of these to transparent. Does that cover this use case or was there another use case you are thinking of?

**Pull request merged**